File: src/p1304/java/solution.java
class Solution {
    public int[] sumZero(int n) {
        int[] result = new int[n];
        if (n % 2 != 0) {
            result[--n] = 0;
        }
        while (n > 0) {
            result[n - 1] = n;
            result[n - 2] = -n;
            n -= 2;
        }
        return result;
    }
}

File: src/p1304/golang/solution.go
func sumZero(n int) (result []int) {
	if n%2 != 0 {
		result = append(result, 0)
		n--
	}
	for n > 0 {
		result = append(result, n, -n)
		n -= 2
	}
	return result
}
